src/serial.rs

```diff
@@ -270,7 +274,8 @@ impl<T: Trigger, EV: SerialEvents, W: Write> Serial<T, EV, W> {
             modem_control: DEFAULT_MODEM_CONTROL,
             modem_status: DEFAULT_MODEM_STATUS,
             scratch: DEFAULT_SCRATCH,
-            in_buffer: VecDeque::new(),
+            in_buffer: Default::default(),
+            out_written: Default::default(),
```

Comment:
No particular reason, just so that there is less stuff to update in case one wanted to change type for these fields (say from VecDequeue to Vec). I'll use explicit defaults.